qa/tasks/cephfs/test_quota.py

```diff
@@ -104,3 +104,13 @@ def test_remote_update_write(self):
         with self.assertRaises(CommandFailedError):
             self.mount_b.write_n_mb("subdir_data/file", 40)
 
+    def test_human_readable_quota_values(self):
+        self.mount_a.run_shell(["mkdir", "subdir"])
+        self.assertEqual(self.mount_a.getfattr("./subdir",
+                                               "ceph.quota.max_bytes"), None)
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "1000")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "1000")
+        with self.assertRaises(CommandFailedError):
+            self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "m10K")
```

Comment:
you might as well add:
- a good test case, e.g. "1g"
- a missing prefix, e.g. "m"
- an empty string

Reply:
@dparmar18 ... and a small note in the docs about the availability of this feature

Reply:
i just added all of them and it is stuck at emty string. need to look whats going on there.

Reply:
> i just added all of them and it is stuck at emty string. need to look whats going on there.

you've probably missed testing for empty string as input

Reply:
> you've probably missed testing for empty string as input

PTAL, added all cases https://github.com/ceph/ceph/pull/49008/commits/c946f9ebb5410211ad4a1c42a415ce2106af3bfa